### PR TITLE
Update test case of reading nested unsigned parquet file

### PR DESCRIPTION
**Pull request by res-life**
Update test case of reading nested unsigned parquet file after cudf fixed issue 9240
This is related to https://github.com/rapidsai/cudf/issues/9240

Signed-off-by: Chong Gao <res_life@163.com>

### CODE REVIEW

**Comment by res-life** [comment]
build

File: tests/src/test/scala/com/nvidia/spark/rapids/ParquetScanSuite.scala [hunk 1]
```diff
@@ -130,9 +129,36 @@ class ParquetScanSuite extends SparkQueryCompareTestSuite {
...
+   *|-- g: struct (nullable = true)
+   *|    |-- c1: short (nullable = true)
+   *|    |-- c2: integer (nullable = true)
+   *|    |-- c3: long (nullable = true)
+   *|-- m1: map (nullable = true)
+   *|    |-- key: short
+   *|    |-- value: short (valueContainsNull
+   *|-- m2: map (nullable = true)
+   *|    |-- key: integer
+   *|    |-- value: integer (valueContainsNu
```
Comment (jlowe) [inline tests/src/test/scala/com/nvidia/spark/rapids/ParquetScanSuite.scala:148]: This and many other lines are truncated

**Comment by res-life** [comment]
build

**Comment by res-life** [comment]
Updated.

**Review by jlowe — APPROVED** [review]

**Pull request merged**